cmd/nephe-controller/main.go

```diff
@@ -47,6 +56,124 @@ func init() {
 	// +kubebuilder:scaffold:scheme
 }
 
+// Secret validator
+type secretValidator struct {
+	client  controllerclient.Client
+	decoder *admission.Decoder
+}
+
+// allowSecretUpdate returns nil only when secret name/namespace/key are same.
+func (v *secretValidator) allowSecretUpdate(new *corev1.Secret, old *corev1.Secret, key string) error {
+	setupLog.Info("Checking secrets are changed")
```

Comment:
shouldnt be using setupLog

Reply:
Added a new logger instance for secret webhook.